### PR TITLE
 Symfony Flex Recipe for Woleet-Bundle

### PR DESCRIPTION
**Pull request by germainlecourtoissloop**
| Q             | A
| ------------- | ---
| License       | MIT

Recipe for https://github.com/Monsieur-SLOOP/woleet-bundle
API for woleet.io blockchain service https://www.woleet.io/

### CODE REVIEW

File: monsieursloop/woleet-bundle/1.0/config/packages/monsieur_sloop_woleet.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+monsieur_sloop_woleet:
+    #API KEY
+    api_key: "YOUR_API_KEY"
+    #API CALLBACK KEY
+    api_callback_key: "YOUR_API_CALLBACK_KEY"
+    #Optional custom API URL
+    #api_url: "https://custiom.api.woleet.io"
```
Comment (Nyholm) [inline monsieursloop/woleet-bundle/1.0/config/packages/monsieur_sloop_woleet.yaml:7]: This is not needed in the recipe. Add a link to your documentation instead.

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you. Just a minor thing

**Comment by germainlecourtoissloop** [comment]
Done @Nyholm  :)

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**